dataeng/resources/snowflake-user-retirement-status-cleanup.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+set -ex
+
+# Setup
+cd $WORKSPACE/analytics-tools/snowflake
+make requirements
+
+python retirement_cleanup.py \
+    --key_path $WORKSPACE/analytics-secure/$KEY_PATH \
+    --passphrase_path $WORKSPACE/analytics-secure/$PASSPHRASE_PATH \
```

Comment:
I'm guessing this is a common pattern, but I'm curious if `PASSPHRASE_PATH` could be more descriptive.